### ISSUE TITLE
Since v10.0.0 supertape no longer prints ok for passed tests

### ISSUE THREAD

**Issue by cloudrac3r** [coderaiser/supertape#8]
The problem seems to only happen with `--format tap` or `--format short`.

supertape version 9.5.0:

```
$ supertape --no-check-assertions-count --format tap test/test.js
TAP version 13
# migrate: migration works
ok 1 it did not throw an error
# migrate: migration works the second time
ok 2 it did not throw an error
# orm: select: get works
ok 3 should equal
```

supertape version 10.0.0 and 10.2.0:

```
$ supertape --no-check-assertions-count --format tap test/test.js
# migrate: migration works
# migrate: migration works the second time
# orm: select: get works
```

I expected the output to have lines starting with  `ok`.

**Comment by coderaiser** [comment]
Looks like it is related with worker thread that make ui more robust, could you please try to pass `—-no-worker` flag is it works this way?

**Comment by coderaiser** [comment]
Just fixed in [v10.2.1](https://github.com/coderaiser/supertape/releases/tag/v10.2.1) 🎉. Is it works for you?

**Comment by cloudrac3r** [comment]
Thanks so much, it works for me!
